### PR TITLE
The inner mechanics of git

### PR DESCRIPTION
**Pull request by hallon-heyman**

### CODE REVIEW

File: contributions/presentation/week2/hheyman-nacca/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Our intention is to to give our viewers a deeper understanding of git beyond the more basic commands-and-branches understanding, such as the abstract object model that drives the functionality.
+ 
+ Slide draft
+ - Short history of git (also including purpose)
+ - Why git is so succesfull
+ - Branching
+ - Object-model in the deeper mechanics of git
+ - Why inner mechanics matter
+
```
Comment (cesarsotovalero) [inline contributions/presentation/week2/hheyman-nacca/README.md:21]: Your proposal is more suitable for an essay. Consider change it!

**Review by cesarsotovalero — CHANGES_REQUESTED** [review]

File: contributions/presentation/week2/hheyman-nacca/README.md (hunk 1 above)
Comment (hallon-heyman) [reply contributions/presentation/week2/hheyman-nacca/README.md:21]: Upon reflection I agree, however, my partner for this assignment already has another partner for her essay assignment so I would ask that we can do a good presentation on this

**Comment by cesarsotovalero** [comment]
Hi @hallon-heyman, I'm closing this PR. Looking forward to your next proposal!

**Comment by hallon-heyman** [comment]
I've spent 3 days just trying to get started on an assignment, this is complete bullcrap.

**Pull request closed without merging**